### ISSUE TITLE
Omni completion doesn't work: complete/completion mismatch

### ISSUE THREAD

**Issue by rnmmnen** [ryyppy/vim-rescript#56]
`vim-rescript` tries to use `complete`:

https://github.com/rescript-lang/vim-rescript/blob/faeeb2deef1ad0ec67fc892ceeb9949f3bf2d77c/autoload/rescript.vim#L346

The underlying `rescript-editor-analysis.exe` expects `completion`:

https://github.com/rescript-lang/rescript-vscode/blob/master/analysis/src/Cli.ml#L66

**Comment by ryyppy** [comment]
Yeah that was a legit change from this commit which I didn't notice until now:
https://github.com/rescript-lang/rescript-vscode/commit/d8183100dc58bb9c5efd3711c1fb3eae301ffa80

Since we are vendoring the language server right now, we may even change this right after updating to the latest LSP.
